tests/__snapshots__/test_help_messages.ambr

```diff
@@ -17,6 +17,7 @@
   ╭─ Commands ───────────────────────────────────────────────────────────────────╮
   │ app         Manages a Snowflake Native App                                   │
   │ connection  Manages connections to Snowflake.                                │
+  │ git         Manages git repositories in Snowflake.                           │
```

Comment:
Can we add test that asserts it is not visible if disabled? We can do this by mocking environment variable, WDYT?

Reply:
Done